### DISCUSSION TITLE
Eufy Smart lock C220 is showing up odd in HA as well as not being able to lock or unlock

### DISCUSSION THREAD

**Discussion by jakob598** [fuatakgun/eufy_security#1121] [Q&A]
So i am new to HA in general. I followed the instructions on how to add the add-on/integration and was able to link my eufy account and the lock did "show" up. I see this device in supported devices so i am unsure of why it is doing this. Please have some patience as i am again new to HA. Thank you all in advance
![github ss1](https://github.com/fuatakgun/eufy_security/assets/164067107/85f2cc4b-6a8a-4d16-a7e0-f9ae22108d06)
![github ss2](https://github.com/fuatakgun/eufy_security/assets/164067107/ed584c21-8c83-469b-b5c3-acfc25809a2a)

**Comment by thomasdg** [comment]
My guess it is is the same as in the post "Update to Bropat 3.0". Those are new devices supported in the version 3.0 of the eufy-security-ws module from bropat. The integration by fuatakgun is flexible enough to recognise the new devices, but bropat added some new functionality in his code which this integration doensn't know how to handle yet. Let's just hope fuatakgun soon has time to add them :)

**Reply by fuatakgun** [reply]
What is the current status after the latest release from bropat?

**Reply by thomasdg** [reply]
It shows my smartdrop as shown by the OP above, but the "new" commands are not usable. Screenshot of the main binary_sensor that appears. There are also some numbers and select states in HA which contain the settings of the smartdrop. Guess the C220 has a similar status at the moment...

<img width="1467" alt="Screenshot 2024-03-23 at 19 51 53" src="https://github.com/fuatakgun/eufy_security/assets/5465002/dd5249c0-a6ee-491d-b643-1c1712a19fbc">
